pkg/detectors/github/github.go

```diff
@@ -94,10 +97,28 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 							s1.ExtraData["username"] = userResponse.Login
 							s1.ExtraData["url"] = userResponse.UserURL
 							s1.ExtraData["account_type"] = userResponse.Type
-							s1.ExtraData["site_admin"] = fmt.Sprintf("%t", userResponse.SiteAdmin)
-							s1.ExtraData["name"] = userResponse.Name
-							s1.ExtraData["company"] = userResponse.Company
-							s1.ExtraData["scopes"] = res.Header.Get("X-OAuth-Scopes")
+							if userResponse.SiteAdmin {
```

Comment:
This is going to be false 99.9% of the time so I don't think it's worth including.